src/traffic_server/traffic_server.cc

```diff
@@ -2007,6 +2036,10 @@ main(int /* argc ATS_UNUSED */, const char **argv)
 
   REC_ReadConfigInteger(thread_max_heartbeat_mseconds, "proxy.config.thread.max_heartbeat_mseconds");
 
+#if TS_USE_LINUX_IO_URING
+  configure_io_uring();
+#endif
+
```

Comment:
Below on line 2076 it is going to print that it is "Using io_uring for AIO" if the server is running in the thread mode.

```
[May  8 14:20:48.641] traffic_server WARNING: <AIO.cc:241 (ink_aio_init)> Using thread for AIO
[May  8 14:20:48.641] traffic_server NOTE: <traffic_server.cc:2076 (main)> Using io_uring for AIO
[May  8 14:20:48.641] traffic_server NOTE: <traffic_server.cc:2080 (main)> io_uring: WQ workers - bounded = 128, unbounded = 256574
```